### ISSUE TITLE
`CitronTokenCode` should be public

### ISSUE THREAD

**Issue by dabrahams** [roop/citron#35]
Or better yet, there should be a way to request public accessibility for any/all of the components in the generated Swift code.  Not having this limits usability.
